site/en/capabilities/index.njk

```diff
@@ -0,0 +1,210 @@
+{% from 'macros/landing-section-usecases.njk' import landingSectionUseCases with context %}
+
+{{ landingSection(
+  'Capabilities, aka. Project Fugu',
+  'Learn about the cross-company effort around enabling the most amazing apps on the web.',
+  sections.intro,
+  'green',
+  'top-3',
+  true,
+  'image/8WbTDNrhLsU0El80frMBGE4eMCD3/lauDAddiUZqlgicYTyYV.svg',
+  '',
+  333,
+  165,
+  'image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/kdPhkPXy7R0mmcJaKyoi.svg'
+  )
+}}
+
+{{ landingSection(
+  'What do developers build with Fugu APIs?',
+  'See what people have built: browse the Project Fugu API Showcase and read about Photoshop on the web!',
+  sections.links,
+  'blue',
+  'top-2',
+  true,
+  'image/8WbTDNrhLsU0El80frMBGE4eMCD3/8FZcBmFowbDKWxpkOytx.jpg',
+  'Fugu fish swarm.',
+  964,
+  289,
+  'image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/kdPhkPXy7R0mmcJaKyoi.svg'
+  )
+}}
+
+{{ landingSection(
+  'Powerful APIs',
+  'Read up on some of the most powerful APIs that enable new use cases not possible before.',
+  sections.powerful_apis,
+  'green',
+  'top-3',
+  true,
+  'image/8WbTDNrhLsU0El80frMBGE4eMCD3/luZO0VUIF4eGhPr4KfIb.svg',
+  '',
+  150,
+  150,
+  'image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/kdPhkPXy7R0mmcJaKyoi.svg'
+  )
+}}
+
+{{ landingSection(
+  'Hardware APIs',
+  'Learn about the hardware APIs we work on in the context of Project Fugu that allow you to access physical devices from the web.',
+  sections.hardware,
+  'yellow',
+  'left',
+  true,
+  'image/8WbTDNrhLsU0El80frMBGE4eMCD3/AZza6zt6XdpzWKySovAb.jpg',
+  'Arduino board and many electronic components.',
+  400,
+  600,
+  'image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/kdPhkPXy7R0mmcJaKyoi.svg'
+  )
+}}
+
+{{ landingSection(
+  'Case studies',
+  'Learn how leading developers have used Fugu APIs to build innovative web experiences.',
+  sections.case_studies,
+  'red',
+  'top-3',
+  true,
+  'image/8WbTDNrhLsU0El80frMBGE4eMCD3/hj30wCIWiTvQSxNI9g5i.svg',
+  '',
+  150,
+  150,
+  'image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/kdPhkPXy7R0mmcJaKyoi.svg'
+  )
+}}
+
+{{ landingSection(
+  'Fugu talk videos',
+  'Watch some talks about Project Fugu APIs to get a better understanding of the kind of apps we want to enable.',
+  sections.videos,
+  'blue',
+  'right',
+  true,
+  'image/8WbTDNrhLsU0El80frMBGE4eMCD3/j5ZqHuEoD6OlA5gRK3IF.svg',
+  '',
+  150,
+  150,
+  'image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/kdPhkPXy7R0mmcJaKyoi.svg'
+  )
+}}
+
+<div class="landing-section landing-section--boxes rounded-lg width-full display-grid gap-top-500">
+  {% for item in sections.resources %}
+    <div class="rounded-lg pad-300 md:pad-400 width-full hairline display-flex direction-column">
+      <h2 class="type--h4">
+        <a class="surface display-inline-flex color-text" href="{{item.url}}">
+          {{ item.title }}
+        </a>
+      </h2>
+      <p class="flex-1">{{ item.description }}</p>
+    </div>
+  {% endfor %}
+  {{ landingDeco(
+    'Additional resources',
```

Comment:
I think you need to place this heading first, there is CSS that will place is visually last.